packages/blade/src/components/SkipNav/_KitchenSink.SkipNav.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
```

Comment:
We can remove this one, unless the nav link is focused it won't show up and anyway it's just a link only.